src/model_repository_manager/model_lifecycle.cc

```diff
@@ -799,10 +802,20 @@ ModelLifeCycle::OnLoadFinal(
     // Mark current versions ready and track info in foreground
     for (auto& loaded : load_tracker->load_set_) {
       std::lock_guard<std::mutex> curr_info_lk(loaded.second->mtx_);
-
       loaded.second->state_ = ModelReadyState::READY;
       loaded.second->state_reason_.clear();
-
+      auto reporter = loaded.second->model_->MetricReporter();
+      const uint64_t now_ns =
+          std::chrono::duration_cast<std::chrono::nanoseconds>(
+              std::chrono::steady_clock::now().time_since_epoch())
+              .count();
+      uint64_t time_to_load_ns = now_ns - loaded.second->load_start_ns_;
```

Comment:
should these be included in the `ifdef TRITON_ENABLE_METRICS` block?

Reply:
Updated fdef TRITON_ENABLE_METRICS block.
Resolved